src/services/player/autoPlayer.js

```diff
@@ -0,0 +1,36 @@
+// export default
+class AutoPlayer {
+  constructor() {
+    this.id = 'AutoPlayer';
+  }
+
+  generateAutoPlayerAnswer(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1) + min);
+  }
+
+  getAnswer(image, answers) {
+    // console.log(image);
+    // console.log(answers);
+    if (typeof image !== 'string') {
+      throw new Error('image type error');
+    }
+    if (Array.isArray(answers) == false) {
```

Comment:
you can leave it just as `if (Array.isArray(answers))` 
coercion automatically implicit here one data type (array) to another (boolean) 


otherwise if we would like to avoid array types in if we would do `if (!Array.isArray(answers)` -> it will implicit if `answers` are array, but will flip true to false or false to true

Reply:
Changed syntax to (!Array.isArray(answers)).
I want to raise an error if it's not an array.